android-pay/build.gradle

```diff
@@ -38,7 +38,8 @@ dependencies {
     javadocDeps 'com.android.support:appcompat-v7:' + android_support_version
     provided 'javax.annotation:jsr250-api:1.0'
     provided 'com.google.android.gms:play-services-wallet:11.0.1'
-    provided 'com.stripe:stripe-android:4.1.3'
+    // This should be changed back to a "provided" line before release
+    compile project(':stripe')
```

Comment:
We could consider adding flag that we build into the a linter that runs before release. Maybe [Release-Blocking] so we programmatically enforce these things.